packages/polaris-viz/src/components/SimpleBarChart/utilities.ts

```diff
@@ -21,10 +21,6 @@ export function getLongestTrendIndicator(
       for (const [index, trend] of trendEntries) {
         const dataPoint = seriesData[index];
 
-        if (trend.value == null) {
```

Comment:
This was just added in https://github.com/Shopify/polaris-viz/pull/1560/files#diff-9ac33a141fa88f91f43ed900a667ec4876d0d408b5af8685ae07d6c646045f16R23-R27 to prevent an undefined value from crashing the SimpleBarChart

Not sure we want to remove it.

Reply:
@pbojinov good observation! This PR also updates `estimateTrendIndicator()` to support an undefined `value` parameter, in which case it uses the [NO_VALUE_WIDTH](https://github.com/Shopify/polaris-viz/blob/40c76ea06b1e5bfd3a51ccb6225a77fbd798ab8d/packages/polaris-viz/src/components/TrendIndicator/constants.ts#L4) constant as a fallback width (`NO_VALUE_WIDTH` is used [in TrendIndicator.tsx](https://github.com/Shopify/polaris-viz/blob/40c76ea06b1e5bfd3a51ccb6225a77fbd798ab8d/packages/polaris-viz/src/components/TrendIndicator/TrendIndicator.tsx#L44) as the width for trend indicators that don't have a value).

Data points with an undefined `value` appear to be working without issue in the Storybook for this PR: https://6062ad4a2d14cd0021539c1b-udbtypmgrc.chromatic.com/?path=/docs/polaris-viz-charts-simplebarchart--with-trend-indicators

Reply:
@pbojinov I added a new Chromatic story to make sure that this behavior is represented: e59b026